docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/kubernetes.md

```diff
@@ -168,7 +168,7 @@ Environment variables:
       - name: OTEL_RESOURCE_ATTRIBUTES
         value: application=my-app,key=value
  ```
-
+   {@import ../../../../reuse/otel-deployment-environment-tag.md}
```

Comment:
Please put it as a part of `OTEL_RESOURCE_ATTRIBUTES`. It's not separated environment variable.